src/main/java/com/example/demo/models/Image.java

```diff
@@ -0,0 +1,116 @@
+package com.example.demo.models;
+
+import jakarta.persistence.*;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table(
+        name = "image",
+        uniqueConstraints = {
+                @UniqueConstraint(name = "url_unique", columnNames = "url")
+        }
+)
+
+public class Image {
+    @Id
+    @SequenceGenerator(
+            name = "image_sequence",
+            sequenceName = "image_sequence",
+            allocationSize = 1
+    )
+    @GeneratedValue(
+            strategy = GenerationType.SEQUENCE,
+            generator = "image_sequence"
+    )
+    private Long imageId;
+    @Column(nullable = false)
```

Comment:
I would consider moving each field here to a separate line along with its annotations as it would make things more readable